### DISCUSSION TITLE
galaxy brain badge

### DISCUSSION THREAD

**Discussion by 772library772** [community/community#35843] [Other Feature Feedback, Questions, & Ideas] [answered]
how do i get the galaxy brain badge?

**Comment by ruhanpandit** [comment, marked as answer]
you have to answer 2 questions

**Comment by raelsei** [comment]
i agree with @octosyntax 🚀

**Reply by mdAshik922** [reply]
how can I get it
Hopefully, there is some resource that can give me that.

**Reply by radar-blip** [reply]
[Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by HalemoGPA** [comment]
You have to answer 2 questions. These 2 questions **must** have at list one up or one react
btw I hope you react to my answer to get it :)

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
